src/test/key_tests.cpp

```diff
@@ -300,6 +300,13 @@ BOOST_AUTO_TEST_CASE(bip340_test_vectors)
         // Verify those signatures for good measure.
```

Comment:
Is 9afa2c9e50370b2918377f3f3eac0950a4296ec0 needed, I think this is just a duplicate, because `key.SignSchnorr` is implicitly doing the same thing.

Reply:
I don't think there is any harm having a test for both and have a slight preference for keeping it this way only because if someone were to change `CKey::SignSchnorr` in the future (e.g., to not use `KeyPair` or remove the method entirely), `KeyPair` would end up untested.